.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: go
+
```

Comment:
how about to add following code to make environment latest?
```
before_install:
  - sudo apt-get update -qq
```

Reply:
done